### PR TITLE
Support for ubuntu 16.04

### PR DESCRIPTION
**Pull request by Zharktas**
Added support for ubuntu 16.04 in Vagrantfile, build scripts and ansible.

Tested it once on virtualmachine with official ckan package install instructions and it did work without issues.

### CODE REVIEW

**Comment by JJediny** [comment]
LGTM

File: common/tmp/after_web.j2 [hunk 1]
```diff
@@ -21,9 +21,11 @@ then
 	/usr/lib/ckan/default/bin/paster make-config ckan /etc/ckan/default/production.ini
 fi
 
+{% if ansible_distribution_version == "12.04"  or ansible_distribution_version == "12.04" %}
```
Comment (amercader) [inline common/tmp/after_web.j2]: This looks like a typo?

Comment (Zharktas) [inline common/tmp/after_web.j2]: yup, typo. fill fix asap

**Comment by amercader** [comment]
Thanks @Zharktas!

**Pull request merged**